Documentation/api.md

```diff
@@ -394,6 +394,7 @@ PrometheusSpec is a specification of the desired behavior of the Prometheus clus
 | priorityClassName | Priority class assigned to the Pods | string | false |
 | portName | Port name used for the pods and governing service. This defaults to web | string | false |
 | arbitraryFSAccessThroughSMs | ArbitraryFSAccessThroughSMs configures whether configuration based on a service monitor can access arbitrary files on the file system of the Prometheus container e.g. bearer token files. | [ArbitraryFSAccessThroughSMsConfig](#arbitraryfsaccessthroughsmsconfig) | false |
+| honorTimestamps | HonorTimestamps configures prometheuses to honor timestamps coming from metrics | *bool | false |
```

Comment:
```suggestion
| honorTimestamps | HonorTimestamps configures Prometheus to honor timestamps coming from metrics | *bool | false |
```

I would rephrase this description because this is configurable. Something like:
> ... configures whether should honor timestamps....